packages/backend/src/clients/Aws/s3.ts

```diff
@@ -0,0 +1,56 @@
+import * as AWS from 'aws-sdk';
+import { LightdashConfig } from '../../config/parseConfig';
+import Logger from '../../logger';
+
+type S3ServiceDependencies = {
+    lightdashConfig: LightdashConfig;
+};
+
+export class S3Service {
```

Comment:
Should we have a `screenshotModel` which abstracts away the s3 config and local storage config? So the part that says `if not s3 then use local` could reside there. And using the service would just be `screenshotModel.create(image)` or `screenshotModel.get()`

Reply:
That logic is currently on the UnfurlService, and not the S3Service, which is only the client with the S3 config. We can have another abstraction for screenshotModel that is used in UnfurlService, but considering this screenshot is not used anywhere else, I think is easier to keep it within UnfrulService, and private. We can still make a ticket for it but I prefer to keep it separate from this PR